lib/execution/events-websocket.ts

```diff
@@ -29,10 +29,10 @@ import {logger} from '../logger.js';
 import {PropertyGetter} from '../properties.interfaces.js';
 
 export class EventsWsBase {
-    protected expectClose: boolean = false;
+    protected expectClose = false;
```

Comment:
I personally would prefer the added type

Reply:
I don't have a strong opinion either way, but consistency is important - and virtually all existing CE code doesn't explicitly specify types that are obvious from assignment.

Reply:
I agree with Ofek here